packages/notification-center/src/components/notification-center/components/notification-item/NotificationListItem.tsx

```diff
@@ -167,3 +248,44 @@ const TimeMark = styled.div<{ novuTheme: INovuTheme; unseen?: boolean }>`
       ? novuTheme?.notificationItem?.unseen?.timeMarkFontColor
       : novuTheme?.notificationItem?.seen?.timeMarkFontColor};
 `;
+
+const NotificationContentContainer = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 10px;
+`;
+
+const AvatarContainer = styled.div`
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 40px;
+  border: 1px solid ${colors.B40};
+  overflow: hidden;
+`;
+
+const NotificationTextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+`;
+
+const SystemIconWrapper = styled.div<{ bgColor: string }>`
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+  background-color: ${({ bgColor }) => `${bgColor}15`};
```

Comment:
`${bgColor}15` probably the `15` should be removed I guess

Reply:
actually we need the `15` here as an `alpha` value.
This creates a light shaded backgroundColor from the foreground color

Reply:
I'm not sure if I would expect this to be defined here and it might confuse after some time. Maybe it would be better if this will be moved to the `avatarSystemIcons` then?